plugins/e57/io/E57Reader.cpp

```diff
@@ -235,143 +134,120 @@ QuickInfo E57Reader::inspect()
     Dimension::IdList dims = layout->dims();
     for (auto di = dims.begin(); di != dims.end(); ++di)
         qi.m_dimNames.push_back(layout->dimName(*di));
-    qi.m_pointCount = m_pointCount;
+    qi.m_pointCount = e57plugin::numPoints(*m_data3D);
```

Comment:
Have you checked that this works?  I don't think m_data3d would be populated.  You need to call initialize() if you want that behavior in inspect().

Reply:
initialize() is already called before accessing `m_data3D`.

Reply:
Sorry .  That code got rolled up in github.